jpo-ode-svcs/src/main/java/us/dot/its/jpo/ode/importer/ImporterDirectoryWatcher.java

```diff
@@ -29,7 +29,7 @@
 public class ImporterDirectoryWatcher implements Runnable {
    
    public enum ImporterFileType {
-      OBU_LOG_FILE
+      LEAR_LOG_FILE, JSON_FILE
```

Comment:
Do we support JSON files now?

Reply:
Not now but we might in the future. I added it because I needed an alternate value for negative Unit Testing.

Reply:
Gotcha, makes sense.